core/python_utils_test.py

```diff
@@ -26,6 +26,7 @@
 import sys
 import tempfile
 import unittest
+import urllib
```

Comment:
Why is this added?

Reply:
@vojtechjelinek  The inbuild function is located within the `urllib` library. The alternative way is to use the `urllib.parse.quote` , should I use that instead?

Reply:
nvm , i think i found the fix, working on it now

Reply:
The thing is that the import is not needed here.

Reply:
@vojtechjelinek  Yah my bad, I removed it

Reply:
@vojtechjelinek  Done